packages/oauth/src/state-stores/spec-utils.ts

```diff
@@ -53,11 +53,18 @@ export class StateStoreChaiTestRunner {
         it('should detect multiple consumption', async () => {
           const { stateStore } = this;
           const installUrlOptions = { scopes: ['channels:read'] };
-          for (let i = 0; i < 200; i++) {
+          for (let i = 0; i < 100; i++) {
             // generate other states
-            await stateStore.generateStateParam(installUrlOptions, new Date());
+            const date = new Date();
+            await stateStore.generateStateParam(installUrlOptions, date);
+            console.log('\tgenerateStateParam:', i, 100, date);
```

Comment:
question: Is this `100` the same as the loop above? If so, it may be good to put it into a variable and document why we chose 100 to avoid a future regression.